src/plugins/intel_npu/src/compiler_adapter/include/driver_compiler_utils.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Should we merge this in driver_compiler_adapter.hpp? Private methods of DriverCompilerAdapter?

Reply:
done